docs/manifest.json

```diff
@@ -1021,6 +1021,11 @@
         "title": "Development",
         "items": [
           [
+            {
+              "title": "Keyless Mode (beta)",
```

Comment:
Shouldn't we remove the `(beta)` since we'll have the `tag` already?

Reply:
You're right! Fixed: https://github.com/clerk/clerk-docs/pull/1699#:~:text=6cc82f9-,fix,-6b6dbf2